### PR TITLE
Revolver Resprites: Imaginos Edition

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
Self Explanatory! This PR resprites every revolver in the game and adds a new variety: The Syndicate Revolver!
image

This also takes into account detective revolvers and their reskins!
image

The Syndicate Revolver itself is a cosmetic change of the regular .357 revolver, which can still be found via gatfruit, or Garbage Day. The Syndicate Revolver can be obtained via the battlecruiser, or any uplink.

Special thanks to @axietheaxolotl for making the Syndicate Revolver sprite.
## Why It's Good For The Game
The previous sprites were either too old, or looked like plastic toys, and desperately needed a visual overhaul. This PR addresses that issue in a very direct manner!
## Changelog
:cl: PositiveEntropy, AxieTheAxolotl
imageadd: Resprites every revolver!
imageadd: Adds the Syndicate Revolver, a visually new, but mechanically identical .357 revolver to uplinks!
/:cl:

### CODE REVIEW

**Comment by Overuseddart** [comment]
Which one is the revolver

**Comment by Imaginos16** [comment]
> Which one is the syndicate revolver

image

**Comment by Overuseddart** [comment]
> > Which one is the syndicate revolver
> 
> image

The sprites are very good, but the syndicate revolver looks like a shotgun pistol to me, maybe just an issue with my perception of it?

**Comment by Imaginos16** [comment]
> The sprites are very good, but the syndicate revolver looks like a shotgun pistol to me, maybe just an issue with my perception of it?

Oh don't worry, there's a shotgun pistol in the works that looks completely different!

**Comment by ChungusGamer666** [comment]
Too high tech, me no likey

**Comment by Mey-Ha-Zah** [comment]
new sprites > soul

**Comment by DAKKA-WAAAGH** [comment]
**Syndicate NERF Revolver**™️

**Comment by Overuseddart** [comment]
Description for the new syndicate revolver is a bit weird, maybe something like “A modern take on a brutal syndicate revolver that has 7 chambers and uses .357 magnum rounds, produced by the Golex Marauders no russian”? My description probably sucks as well but I think the description could use a change in relation to the sprite.

**Comment by zxaber** [comment]
> > Which one is the syndicate revolver
> 
> image

Looks like a tacticool flare gun to me, I think the barrel is too chunky. Looks nice otherwise.

**Review by Twaticus — APPROVED** [review]

**Comment by Bawhoppen** [comment]
Syndicate revolver looks like a laser pistol or something, rather than just a regular revolver. Maybe tone down the red's brightness?

File: code/modules/uplink/uplink_items/dangerous.dm [hunk 1]
```diff
@@ -95,8 +95,8 @@
 
 /datum/uplink_item/dangerous/revolver
 	name = "Syndicate Revolver"
-	desc = "A brutally simple Syndicate revolver that fires .357 Magnum rounds and has 7 chambers."
-	item = /obj/item/gun/ballistic/revolver
+	desc = "A modernized Syndicate revolver by Waffle Co. that fires brutal .357 Magnum rounds and has 7 chambers."
```
Comment (Fikou) [inline code/modules/uplink/uplink_items/dangerous.dm:98]:
```suggestion
	desc = "A modernized Syndicate revolver by Waffle Co. that fires 7 brutal rounds of .357 Magnum."
```

**Comment by YakumoChen** [comment]
Syndievolver looks really silly. It's also just too big. The barrel is too big and long for a pistol, and over-engineered for one of the simplest gun designs in history.

I don't see the value of changing the appearance from the traditional style either, there's more value in keeping the waters muddy for the revolvers source.

(Also a clown with a reverse revolver can't hide it in a pile of gatfruit guns)

**Comment by Imaginos16** [comment]
> Syndievolver looks really silly. It's also just too big. The barrel is too big and long for a pistol, and over-engineered for one of the simplest gun designs in history.
> 
> I don't see the value of changing the appearance from the traditional style either, there's more value in keeping the waters muddy for the revolvers source.
> 
> (Also a clown with a reverse revolver can't hide it in a pile of gatfruit guns)

Except the clown can do that because I specifically snowflaked reverse revolvers to not look like the syndicate one. This includes gatfruit and garbage day lol.

**Comment by Helg2** [comment]
Why cant you just make it as alternative skin instead of changing everything. And you also didnt show how revolvers looked before change and only showed how it'll look after.

**Comment by Imaginos16** [comment]
> Why cant you just make it as alternative skin instead of changing everything. 

image

**Comment by bluetshirtguy** [comment]
edgy advanced techno guns win again

**Comment by calliophis-maculiceps** [comment]
> edgy advanced techno guns win again

brave spessmen retrofuturism fans reduced to mush by heinous evil artstation trending tab sci fi twitter pixel artists

**Comment by Serxule** [comment]
for the .357: its better than the _other_ .357 resprite, barrel and chamber are def too big and looks like a flare gun as someone else said, but it actually resembles syndicate and doesnt look like an energy weapon

everything else: looks good

**Pull request merged**